sample-code/vc-using-framestream/vc-using-framestream/main.cpp

```diff
@@ -52,16 +54,26 @@ int main(int argc, char **argv) {
 
   const auto & args = argParseResult.args;
 
+  // When multicasting, the cooperating applications will likely want to
+  // choose a receive port in advance so the non-controlling application
+  // knows where to listen for packets.
+  const bool useMulticast = args.useMulticast.has_value() && args.useMulticast.value();
+  const auto receiveFromAddr =
+    useMulticast
+    ? boost::asio::ip::udp::endpoint(boost::asio::ip::address::from_string("239.0.0.42"), 59595)
```

Comment:
Hmm.  Do we want to hardcode the multicast IP and port?  Seems okay for now but at some point I could see users wanting to pass these in as command line args.